src/Controls/tests/UITests/Tests/Issues/Issue774.cs

```diff
@@ -0,0 +1,48 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue774 : _IssuesUITest
+	{
+		public Issue774(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "ActionSheet won't dismiss after rotation to landscape";
+		
+		protected override void FixtureTeardown()
+		{
+			App.SetOrientationPortrait();
+			App.Back();
+
+			base.FixtureTeardown();
+		}
+
+		[Test]
+		[Category(UITestCategories.ActionSheet)]
+		public void Issue774TestsDismissActionSheetAfterRotation()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.iOS, TestDevice.Mac, TestDevice.Windows });
+
+			App.Click("ShowActionSheet");
+			App.Screenshot("Show ActionSheet");
+
+			App.SetOrientationLandscape();
+			App.Screenshot("Rotate Device");
```

Comment:
Should these be VerifyScreenshot?

Reply:
The test is a port from an existing one, but definitely here could have sense.